md/monitoring.md

```diff
@@ -11,38 +11,50 @@ The artificial intelligence algorithm applies a prediction to each case: its lik
 Based on this information, the page shows a pie chart with the proportion of cases *Late* (already behind the deadline), *Predicted late* (likelihood is below 50%), and *On time* (likelihood is 50% or above). 
 The decision threshold that makes cases *Predicted late* or *On time* can be configured in the [Configuration Living Application](configure.md). 
 
-It also contains a tabbed list of cases that identifies each case by its ID as well as the name of the initiator. 
+It also contains a tabbed list of cases that identify each case by its ID as well as the name of the initiator. 
```

Comment:
Why not "identifies" ? Is it not the "list" who identify each case ?

Reply:
Of course, you are right - it's the list. I assumed its the list but got wrong with the conjugation.